.github/workflows/node.js.yml

```diff
@@ -11,12 +11,38 @@ on:
 
 jobs:
   build:
-
+     runs-on: ubuntu-latest
+     steps:
+        - uses: actions/checkout@v2
+        - name: npm install and build webpack
+          run: |
+            npm install
+            npm run build
+  test:
     runs-on: ubuntu-latest
+    
+    
+    strategy:
+      matrix:
+          os: [ubuntu-latest, windows-2016]
+          node-version: [12.x, 14.x]
+      steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install, and test
+        run: |
+          npm install
+          npm test
+        env:
+          CI: true
```

Comment:
- uses: actions/download-artifact@main
      with:
        name: webpack artifacts
        path: public
    - name: npm install, and test